src/debian/snmp/snmp_getdata.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+
+import subprocess,json,os
+
+def run_command(command):
+    result = os.popen(command).read()
```

Comment:
How your script react if there is an error in the command?

Reply:
The run_command function will show the error:
```
# /usr/local/sbin/snmp_getdata.py
/bin/sh: 1: no: not found
```

and the oid will be empty in the /tmp/snmpdata.txt file but it won't impact the rest of the processing:
```
# head /tmp/snmpdata.txt
.2.1:smartctl /dev/sda
.2.1.0:PASSED
.2.2:smartctl /dev/sdb
.2.2.0:PASSED
.2.3:volontary error
.2.3.0:
.1.1:/usr/sbin/crm status
.1.1.1:Status of pacemakerd: 'Pacemaker is running' (last updated 2024-02-26 15:42:09 +01:00)
```

An empty value seems to be ignored by snmp_passpersist so it won't show in snmpwalk.

Anyway, I will had what is needed to log the program output to journalctl
```
*/5 * * * * root /usr/bin/systemd-cat -t "seapathsnmp-cron" /usr/local/sbin/snmp_getdata.py
```